go/vm/lfvm/instructions.go

```diff
@@ -617,7 +617,7 @@ func opSelfbalance(c *context) {
 }
 
 func opBaseFee(c *context) {
-	if c.isLondon {
+	if c.revision == vm.R10_London {
```

Comment:
The flag `isXXX` was set if the revision was `XXX` or higher. So I guess this needs to be 
```suggestion
	if c.revision >= vm.R10_London {
```

Reply:
Just a question:
Changes in the revision are assumed to be valid in further revisions? 
rephrasing: Is the OpCode revision system always incremental?

Reply:
Those changes may be backward incompatible.